### PR TITLE
prototype explanation

### PR DESCRIPTION
**Pull request by pmconnolly80**

### CODE REVIEW

File: front-end-notes/prototypes.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#Prototypal Inheritance
+
+Prototypal inheritance is a feature that allows JavaScript objects to 'inherit' properties from the object used to create them. This is a great way to streamline object production and avoid repetitive code. 
+
+###This example comes from Steve's JS102 Codepen Example
```
Comment (NewEvolution) [inline front-end-notes/prototypes.md]: A link to the codepen would be great.

**Comment by tombler** [comment]
Might be worth mentioning that using the `new` keyword is referred to as a constructor function: 

"A constructor function is used with the new keyword (which causes JavaScript to automatically create a new object, set this within the function to that object, and return the object): var objFromConstructor = new ConstructorFunction();"

File: front-end-notes/prototypes.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Create the second car
+
+``var secondCar = new Car("blue");``
+
+Create the third car
+
+``var thirdCar = new Car("gold");``
+
+#Links:
```
Comment (BrendonPierson) [inline front-end-notes/prototypes.md]: Consider reformatting to decrease loudness?

Comment (Dan-Hodges) [inline front-end-notes/prototypes.md]: We turned it up to 11.

**Pull request merged**